contrib/main.yml.sample

```diff
@@ -145,6 +145,10 @@ worker_instance_count: 3
 # disk is an io1 EBS volume, not instance storage.
 worker_instance_root_disk_size_gb: 64
 
+# worker_instance_ebs_optimized controls whether or not the instance
+# is provisioned as an EBS-optimized instance. This defaults to true.
+#worker_instance_ebs_optimized: true
```

Comment:
Please update the description of worker_instance_type given that its stated requirement is not always true.